server/pbenchinacan/run-pbench-in-a-can

```diff
@@ -1,24 +1,127 @@
 #!/bin/bash
 
+#
+# Deploy a self-contained Pbench server.
+#
+# This script creates a pod for the Pbench Server deployment using
+# three containers: All the dependant services associated with the
+# Pbench Server i.e. Elasticsearch instance, PostgreSQL database instance,
+# and KeyCloak broker instance. After starting the dependent services,
+# this script configures the Keycloak broker, and then starts the Pbench Server.
+#
+# To invoke the pbench-server pod using this script it should run from the
+# root of the Git checkout.
+#
+# The resulting pod exports the following ports:
+#   5432 - PostgreSQL
+#   8090 - keycloak authentication server
+#   9200 - Elasticsearch
+#
+# The resulting pbench-server container runs on the following port
+#   8080 - HTTP web server (and Pbench Dashboard)
+
 export PB_CONTAINER_REG=${PB_CONTAINER_REG:-"images.paas.redhat.com"}
-export PB_ORG_NAME=${PB_ORG_NAME:-"pbench"}
+export PB_CONTAINER_IMAGE_PREFIX="${PB_CONTAINER_REG}/pbench"
 
 export PB_ELASTICSEARCH_IMAGE_NAME=${PB_ELASTICSEARCH_IMAGE_NAME:-"elasticsearch"}
 export PB_ELASTICSEARCH_IMAGE_TAG=${PB_ELASTICSEARCH_IMAGE_TAG:-"7.13.3"}
 export PB_POSTGRESQL_IMAGE_NAME=${PB_POSTGRESQL_IMAGE_NAME:-"postgresql-13"}
 export PB_POSTGRESQL_IMAGE_TAG=${PB_POSTGRESQL_IMAGE_TAG:-"latest"}
 export PB_KEYCLOAK_IMAGE_NAME=${PB_KEYCLOAK_IMAGE_NAME:-"pbenchinacan-keycloak"}
-export PB_KEYCLOAK_IMAGE_TAG=${PB_KEYCLOAK_IMAGE_TAG:-"latest"}
+export PB_KEYCLOAK_IMAGE_TAG=${PB_KEYCLOAK_IMAGE_TAG:-"20.0.3"}
 export PB_SERVER_IMAGE_NAME=${PB_SERVER_IMAGE_NAME:-"pbench-server"}
 export PB_SERVER_IMAGE_TAG=${PB_SERVER_IMAGE_TAG:-"$(cat jenkins/branch.name)"}
 export PB_POD_NAME=${PB_POD_NAME:-"pbench-in-a-can_${PB_SERVER_IMAGE_TAG}"}
 
-export PB_COMMON_IMAGE_PULL_POLICY=${PB_COMMON_IMAGE_PULL_POLICY:-"Always"}
+export PB_COMMON_IMAGE_PULL_POLICY=${PB_COMMON_IMAGE_PULL_POLICY:-"always"}
 export PB_ELASTICSEARCH_IMAGE_PULL_POLICY="${PB_ELASTICSEARCH_IMAGE_PULL_POLICY:-${PB_COMMON_IMAGE_PULL_POLICY}}"
 export PB_POSTGRESQL_IMAGE_PULL_POLICY="${PB_POSTGRESQL_IMAGE_PULL_POLICY:-${PB_COMMON_IMAGE_PULL_POLICY}}"
 export PB_KEYCLOAK_IMAGE_PULL_POLICY="${PB_KEYCLOAK_IMAGE_PULL_POLICY:-${PB_COMMON_IMAGE_PULL_POLICY}}"
 export PB_SERVER_IMAGE_PULL_POLICY="${PB_SERVER_IMAGE_PULL_POLICY:-${PB_COMMON_IMAGE_PULL_POLICY}}"
 
 export PB_DASHBOARD_DIR="${PB_DASHBOARD_DIR:-${PWD}/dashboard/build/}"
 
-envsubst < $(dirname ${0})/pod.yml | podman play kube -
+podman pod create --name ${PB_POD_NAME} -p 9200:9200 -p 5432:5432 -p 8090:8090
```

Comment:
I have an awkward question...this is the pod creation, where is the pod destruction?  And, more to the point, now that the Pbench Server is no longer in the pod, where is the code which tears **_it_** down when we're done with it?

My [comment](https://github.com/distributed-system-analysis/pbench/pull/3216#discussion_r1094840029) above notwithstanding, I think `jenkins/run-server-func-tests` [might need some tweaking](https://github.com/distributed-system-analysis/pbench/pull/3224#issuecomment-1413970060) as a part of this PR?

Reply:
Hmm, Yes the pod destruction happens at the end of `run-server-func-tests` but that will just remove the pod of the dependent service. We need `podman rm` to remove the server container as well.